uber/config.py

```diff
@@ -184,6 +184,11 @@ def DEALER_APPS(self):
         with sa.Session() as session:
             return session.query(sa.Group).filter(sa.Group.tables > 0, sa.Group.cost > 0, sa.Group.status == self.UNAPPROVED).count()
 
+    @request_cached_property
+    @dynamic
+    def ATTENDEE_BADGES_SOLD(self):
+        return self.get_badge_count_by_type(badge_type)
```

Comment:
This looks like a typo?  Should this say ``return self.get_badge_count_by_type(c.ATTENDEE_BADGE)``?

Reply:
Yep, now I'm curious why that didn't throw an error :(